### DISCUSSION TITLE
Added New Feature to network

### DISCUSSION THREAD

**Discussion by dragonzyk** [BornToBeRoot/NETworkManager#2311] [Ideas]
With simulation can create a lot schematic network but using that without experience and alone maybe waste time but in pocket tracer after simulation a lot things like change activity,rules,permission,incoming method begin

**Comment by BornToBeRoot** [comment]
Do you mean something like cisco packet tracer?

**Comment by BornToBeRoot** [comment]
Closed, as not planned!
